PKGBUILD

```diff
@@ -46,9 +46,21 @@ package() {
   install -D -m644 99-displaylink.rules "$pkgdir/etc/udev/rules.d/99-displaylink.rules"
   install -D -m755 udev.sh "$pkgdir/opt/displaylink/udev.sh"
 
-  echo "Installing DLM systemd service"
-  install -D -m644 displaylink.service "$pkgdir/usr/lib/systemd/system/displaylink.service"
-  install -D -m755 displaylink-sleep.sh "$pkgdir/usr/lib/systemd/system-sleep/displaylink.sh"
+  echo "Installing DLM runit service"
+
+  mkdir -p "$pkgdir/etc/runit/sv/displaylink"
+  install -Dm 644 displaylink.run "$pkgdir/etc/runit/sv/displaylink/run"
+  chmod u+x "$pkgdir/etc/runit/sv/displaylink/run"
+
+  echo """
```

Comment:
Do I even need this? I get a generic message from the hook anyway

```
:: Running post-transaction hooks...
(1/1) Displaying runit service help ...
        ==> Add a service:
        ln -s /etc/runit/sv/<service> /run/runit/service/
        ==> Start/stop/restart a service:
        sv <start/stop/restart> <service>
```

Reply:
I agree that you shouldn't need this. I'm not even sure where caveats generally go in PKGBUILDs.